openemrs/config/routes.rb

```diff
@@ -1,3 +1,9 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
```

Comment:
This is an example of how we should have them boilerplate comments — that's if we are to have them at all